src/main/java/net/fabricmc/loom/task/MigrateMappingsTask.java

```diff
@@ -72,7 +75,35 @@ public void doTask() throws Throwable {
 		String outputDir = (String) properties.get("outputDir");
 		if (outputDir == null) outputDir = "remappedSrc";
 
-		File mappings = project.getConfigurations().detachedConfiguration(project.getDependencies().module(properties.get("mappings"))).getSingleFile();
+		File mappings;
```

Comment:
Make this `final`

Reply:
or better, make this whole block a function that returns `File`.

Reply:
Yes, I do think a method for this would be neater.